packages/babel-plugin-proposal-export-default-from/test/fixtures/export-default/default-compound-es6/output.mjs

```diff
@@ -1,3 +1,2 @@
-import _v from "mod";
-export { _v as v };
+export { default as v } from "mod";
 export { x, y as w } from "mod";
```

Comment:
Further optimization idea: we can merge `{ default as v }` with the next specifier if exists.

Reply:
Sure. And here come a case

```JavaScript
export v, * as ns, { x, y as w } from "mod"; 
```

if there is a namespace compound, should we merge it?

```JavaScript 
// a.js
export { default as v,  x, y as w} from "mod";
export * as ns from "mod";
```

The export order will change.
```JavaScript
import * as ns from "a.js";
Object.keys(ns) // order changed
```

Reply:
Ok, it will be merged if there is no namespace export.